specs/tasks.js

```diff
@@ -71,12 +71,16 @@ function processAction(action) {
       if (action.offset !== undefined ) {
         offset = action.offset
       }
-      browser.scroll(selector, null, offset)
+      // Do not scroll on replace action,
+      // selector is not necessarily an element.
+      if (action.replace === undefined) {
```

Comment:
nitpick: I would remove this `if` and move `scroll` call in front of `click` and `setValue` calls, then it will be more readable. And also reorganize if-elses so that replace is first. Then you have better grouping when selectors are changing something or making some browser actions.